util/src/main/java/com/google/cloud/hadoop/util/ApiErrorExtractor.java

```diff
@@ -205,6 +207,15 @@ public boolean rateLimited(IOException e) {
     return false;
   }
 
+  /**
+   * Determines if a given Throwable is caused by Quota Exceeded. Recursively checks getCause() if
+   * outer exception isn't an instance of the correct class.
+   */
+  public boolean quotaExceeded(IOException e) {
```

Comment:
Please, add a test for this method in [ApiErrorExtractorTest](https://github.com/GoogleCloudDataproc/hadoop-connectors/blob/master/util/src/test/java/com/google/cloud/hadoop/util/ApiErrorExtractorTest.java).